src/auth-service/models/Preference.js

```diff
@@ -152,6 +184,11 @@ const PreferenceSchema = new mongoose.Schema(
       },
     ],
     selected_sites: [{ type: siteSchema }],
+    selected_grids: [{ type: gridSchema }],
+    selected_devices: [{ type: deviceSchema }],
+    selected_cohorts: [{ type: cohortSchema }],
+    selected_airqlouds: [{ type: airqloudSchema }],
+
```

Comment:
Hi @FaithDaka , I have just introduced new array fields which take in objects for:

1. grids
2. devices
3. cohorts
4. airqlouds

Each of these objects just expects 2 things for now:

1. _id
2. name

cc: @Codebmk / @OchiengPaul442